### PR TITLE
Fix the bugs in 'Create User' and 'Edit User' endpoints

### PR DESCRIPTION
**Pull request by alperendivriklioglu**
The code in this pull request resolves the issues #232 and #173 . I kindly ask you to review the code.

### CODE REVIEW

**Comment by alperendivriklioglu** [comment]
I've noticed a small bug, so, I close this pull request for now. I'll open it as soon as I push the new code.

**Review by hilal-demir — APPROVED** [review]

File: Platon/backend/app/auth_system/views.py [hunk 1]
```diff
@@ -346,63 +346,61 @@ def put(requester_id, self):
...
-            # If yes, starts acting upon the request.
-            if not any(form.data.values()):
-                return make_response(jsonify({"message" : "Server has received the request but there was no information to be updated."}), 202)
+            # Tries to connect to the database.
+            # If it fails, an error is raised.
+            try:
+                existing_user = User.query.filter_by(id=requester_id)
+            except:
+                return make_response(jsonify({"error" : "The server is not connected to the database."}), 500)
             else:
```
Comment (cakmadam98) [inline Platon/backend/app/auth_system/views.py:355]: why did you use else statement here? Because there is no if statement above

**Review by HUmutOzdemir — APPROVED** [review]

**Review by cakmadam98 — COMMENTED** [review]
I've added some comments.

File: Platon/backend/app/auth_system/views.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def get(requester_id, self):
                                         "e_mail": logged_in_user.e_mail,
                                         "google_scholar_name": logged_in_user.google_scholar_name,
                                         "researchgate_name": logged_in_user.researchgate_name,
-                                        "position": user_position.name,
+                                        "job": user_job.name,
                                         "institution": logged_in_user.institution
```
[the remark this answers is not in the document]
Comment (alperendivriklioglu) [reply Platon/backend/app/auth_system/views.py:168]: Ah, sorry, I did not know that part existed. I'll add it in the next chance I get.

File: Platon/backend/app/auth_system/views.py (hunk 1 above)
Comment (alperendivriklioglu) [reply Platon/backend/app/auth_system/views.py:355]: @cakmadam98 Try-except blocks can have an "else" part too. "Else" in this context means "try, if no exception is raised, continue here."

**Pull request merged**